challenges/Backend Challenge/app.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask
+import requests
+
+app = Flask(__name__)
+arr = []
+
+@app.route('/')
+def index():
+	return "Hello World!"
+
+@app.route('/data', methods = ['GET'])
+def sortnumbers():
+	if len(arr) == 0:
+		return "Number list is empty"
+	arr.sort()
+	res = ""
+	for ele in arr:
+		res += str(ele) + " "
+	return res
+
+@app.route('/data', methods = ['POST'])
+def validatenumbers():
+	response = requests.request('POST', 'http://localhost:5000/data/list.txt')
+    e = response.json()
+    arr = e['numberList']
+    if(len(arr)<500):
```

Comment:
It looks like you check for the list being less than 500 elements. The challenge asks for the list to also not exceed 500 numbers.  Do you also check if the array is greater than 500 numbers?

Reply:
So sorry I missed that! I have fixed it. Please have a look at app1.py. Thank you.